### ISSUE TITLE
A single indentation style would be great.

### ISSUE THREAD

**Issue by andrewchawk** [andrewchawk/coq-tutorial#1]
https://github.com/andrewchawk/coq-tutorial/blob/ecc75011358d7bdb5ecfadd4ddfd84c36953bf20/coq-tutorial.v#L553-L574

**Comment by onestruggler** [comment]
Agreed. Thank you. Sorry for the late reply. I just noticed this. Can you make a pull request (with your other revisons)? Then I can merge it.

**Comment by andrewchawk** [comment]
> Agreed. Thank you. Sorry for the late reply. I just noticed this. Can you make a pull request (with your other revisons)? Then I can merge it.

Hey, my reply was late, too.  In my defense, though, you edited your comment, so I just now noticed that you wanted a pull request.  :-)

I will create the pull request, but I never actually got around to fixing up the indentation.

**Comment by onestruggler** [comment]
Hi, thanks for the pull request. I will merge it asap. I wrote another tutorial about reflection in Agda. Not sure if you are interested.
________________________________
